modules/oe_theme_content_consultation/oe_theme_content_consultation.module

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the Consultation content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\oe_content\CallEntityWrapperInterface;
+use Drupal\oe_content_consultation\ConsultationNodeWrapper;
+
+/**
+ * Implements hook_theme().
+ */
+function oe_theme_content_consultation_theme() {
+  return [
+    'oe_theme_content_consultation_closed_text' => [
+      'variables' => [
+        'text' => '',
+      ],
+    ],
+  ];
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function oe_theme_content_consultation_node_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  if ($entity->bundle() !== 'oe_consultation') {
+    return;
+  }
+
+  if ($build['#view_mode'] === 'full') {
+    // Override field labels.
+    if ($entity->get('oe_departments')->count() === 1) {
+      $build['oe_departments']['#title'] = t('Department');
+    }
+
+    // Show respond to the consultation information based status.
+    $consultation_node_wrapper = ConsultationNodeWrapper::getInstance($entity);
+    $status = $consultation_node_wrapper->getStatus();
+    if ($status === CallEntityWrapperInterface::STATUS_OPEN) {
+      $build['extra_field_oe_consultation_closed_status_text']['#access'] = FALSE;
+      if ($entity->get('oe_consultation_response_button')->isEmpty()) {
+        $build['oe_consultation_response_button']['#access'] = FALSE;
+      }
+    }
+    if ($status === CallEntityWrapperInterface::STATUS_CLOSED) {
+      $build['oe_consultation_guidelines']['#access'] = FALSE;
+      $build['oe_consultation_response_button']['#access'] = FALSE;
+    }
+    if ($status === CallEntityWrapperInterface::STATUS_UPCOMING) {
+      $build['oe_consultation_response_button']['#access'] = FALSE;
+    }
+  }
+}
+
+/**
+ * Prepares variables for in-page navigation.
+ *
+ * Default template: oe-theme-helper-in-page-navigation.html.twig.
+ *
+ * @param array $variables
+ *   An array of variables.
+ */
+function oe_theme_content_consultation_preprocess_oe_theme_helper_in_page_navigation__node__oe_consultation(array &$variables): void {
+  if (count($variables['entity']->get('oe_consultation_contacts')) > 1) {
+    $index = count($variables['items']);
+    $variables['items'][$index - 1]['label'] = 'Contacts';
```

Comment:
Not sure that this approach is reliable enough because page navigation items can be re-ordered.
Let's implement something like:

```
  if (count($variables['entity']->get('oe_consultation_contacts')) > 1) {
    foreach ($variables['items'] as $index => $item) {
      if ($item['label'] === (string) t('Contact')) {
        $variables['items'][$index]['label'] = t('Contacts');
      }
    }
  }
```